db/schema.rb

```diff
@@ -460,6 +469,33 @@
     t.index ["family_id"], name: "index_merchants_on_family_id"
   end
 
+  create_table "messages", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "chat_id", null: false
+    t.uuid "user_id"
+    t.text "content"
+    t.text "log"
+    t.string "role"
+    t.string "status", default: "pending"
+    t.boolean "hidden", default: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["chat_id"], name: "index_messages_on_chat_id"
+    t.index ["user_id"], name: "index_messages_on_user_id"
+  end
```

Comment:
Looks like we're still getting the extra tables here.  You may need a full `rails db:migrate:reset` since `db:reset` will just rebuild based on this existing `schema.rb` and not perform the migrations